packages/web/src/dev/ValidatorsListApp.tsx

```diff
@@ -0,0 +1,87 @@
+import ApolloClient from 'apollo-boost'
+import { InMemoryCache } from 'apollo-cache-inmemory'
+import fetch from 'cross-fetch'
+import gql from 'graphql-tag'
+import getConfig from 'next/config'
+import * as React from 'react'
+import { ApolloProvider, Query } from 'react-apollo'
+import ShowApolloError from 'src/dev/ShowApolloError'
+import ValidatorsList from 'src/dev/ValidatorsList'
+import { I18nProps, withNamespaces } from 'src/i18n'
+
+function createApolloClient() {
+  return new ApolloClient({
+    uri: getConfig().publicRuntimeConfig.LEADERBOARD.uri,
```

Comment:
we can replace `LEADERBOARD` to `BLOCKSCOUT` (also for the leaderboard app)

Reply:
Done, thanks!